src/detect-engine-address-ipv6.h

```diff
@@ -24,11 +24,11 @@
 #ifndef SURICATA_DETECT_ENGINE_ADDRESS_IPV6_H
 #define SURICATA_DETECT_ENGINE_ADDRESS_IPV6_H
 
-int AddressIPv6Lt(Address *, Address *);
-int AddressIPv6Gt(Address *, Address *);
-int AddressIPv6Eq(Address *, Address *);
-int AddressIPv6Le(Address *, Address *);
-int AddressIPv6Ge(Address *, Address *);
+int AddressIPv6Lt(const Address *, const Address *);
```

Comment:
Nice, but why ?

Reply:
Don't see this point of this question. It's just a general cleanup.

Reply:
Why do we want to constify these specific functions ? Any reason why they stood out ?